test/e2e/v2/manager/task.go

```diff
@@ -126,12 +127,233 @@ var _ = Describe("Get and delete task with Manager", func() {
 			done = waitForDone(job, managerPod)
 			Expect(done).Should(BeTrue())
 
-			// Check delete task response is not null.
-			Expect(job.Result).NotTo(BeNil())
+			// Check delete task response is valid.
+			isValid = checkTaskResponse(job)
+			Expect(isValid).To(BeTrue())
```

Comment:
Expect(validateTaskResponse()).To(BeTrue())

Reply:
OK, updated.